inter-operator/architecture/basic.md

```diff
@@ -603,8 +605,21 @@ Schedulers are basically custom controller running on master cluster watching on
 This is just a sample scheduler suitable only for the single cluster setup. In that case, it schedules all the instances in the one cluster which is part of the setup. It is not suitable for the multi-cluster setup.
 #### Round Robin Scheduler
 As the name suggests, round robin scheduler schedules instances in round robin fashion. At this point, it does not take care of capacity and if interoperator restarts, it loses the state about the next cluster to be scheduled and starts scheduling from the beginning. 
-#### Least filled Scheduler
-This scheduler schedules instance in the least filled cluster.
+#### Least Utilized Scheduler
+This scheduler schedules instance in the least utilized cluster.
+#### Label Selector based Scheduler
+Label selector based scheduler chooses clusters for a service instance based on the label selector defined. Label selector is a go template defined within the `SFPlan`, The template can be evaluated to a label selector string which the scheduler uses to choose cluster for the service instance provisioning. An example of such a template can be the following.
+```yaml
+  - action: clusterlabel
```

Comment:
```suggestion
  - action: clusterSelector
```

Also, is `action` the right way to capture this? It is not really an *action*. Besides, is there any possibility to support anything other than `gotemplate` type here?

Reply:
These are certainly not `osb` actions. But I think the current set of actions are also convoluted, with `status` and `sources` template. Type would mostly be `gotemplate` only. One option is use something else other than `templates`.